### PR TITLE
Review of updated UX for initial login/signup

### PR DESCRIPTION
**Pull request by asimkhan73301**
This PR solves issue #140 

**What does this PR do?**

1. Updated Top Languages page to redirect to `/settings` with param `firstTime=True`
2. Updated Basic Info page to redirect to `/feed` on Profile Save if it's first time.

**Any background context you want to provide?**

**Screenshots?**

### CODE REVIEW

**Comment by vipulsnp** [comment]
@asimkhan73301 Update?

**Comment by vipulsnp** [comment]
@irshadjsr21 No updates on this PR.

File: src/components/TopLanguages.jsx [hunk 1]
```diff
@@ -50,7 +50,10 @@ export default function TopOrganisation() {
     db.collection('users').doc(User.uid).update({
       followingLanguages: followed
     });
-    Router.replace('/feed');
```
Comment (pravleen17) [inline src/components/TopLanguages.jsx:53]: router.replace(location, onComplete?, onAbort?)
We should use this format. I think then this problem will be solved.

File: src/components/Settings/BasicInfo.jsx [hunk 2]
```diff
@@ -66,8 +69,12 @@ let uid = null;
 
 
     const response = await setBasicInfo(formData);
-    if(response.status === 'success')
```
Comment (pravleen17) [inline src/components/Settings/BasicInfo.jsx:69]: This line should be removed.

**Pull request closed without merging**